### PR TITLE
use unqualified user and role name to choose resource_groups

### PR DESCRIPTION
**Pull request by satanson**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

when choose a resource_group for a query, we must use unqualified user and role.

### CODE REVIEW

**Review by dirtysalt — DISMISSED** [review]

**Review by laotan332 — DISMISSED** [review]

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 15 / 17 (88.24%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/WorkGroupMgr.java|14|16|87.50%|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|1|1|100.00%|

**Review by ZiheLiu — DISMISSED** [review]

**Review by laotan332 — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/catalog/WorkGroupClassifier.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ public boolean isVisible(String user, String role, String sourceIp) {
         if (this.role != null && !this.role.equals(role)) {
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/catalog/WorkGroupClassifier.java:109]: needn't ignore case here？

**Comment by satanson** [comment]
run starrocks_fe_unittest

**Review by ZiheLiu — APPROVED** [review]

**Review by dirtysalt — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/catalog/WorkGroupClassifier.java (hunk 1 above)
Comment (satanson) [reply fe/fe-core/src/main/java/com/starrocks/catalog/WorkGroupClassifier.java:109]: if role condition is present, then it is must be matched strictly.

**Review by kangkaisen — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 23 / 23 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/catalog/WorkGroupClassifier.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/catalog/WorkGroupMgr.java|21|21|100.00%|

**Pull request merged**